packages/frint-react/src/components/Region.js

```diff
@@ -148,18 +145,14 @@ export default class Region extends React.Component {
 
   componentWillReceiveProps(nextProps) {
     const {
-      name = this.props.name,
-      uniqueKey = this.props.uniqueKey,
-      data = this.props.data,
```

Comment:
@fahad19 do you know if we need these defaults based on the current props? I'm assuming we don't, but maybe there is a reason for it that I'm not aware of 🤔

Reply:
I actually don't see any value in extracted them separately any more.

Reply:
Sure, they will still be present in `...props` but they won't have their default values based on `this.props` anymore. I'm wondering if this change could break something.